### ISSUE TITLE
Create unit tests for MidiTrait

### ISSUE THREAD

**Issue by raphaelDkhn** [cienicera/Koji#28]
Create unit tests for all functions of `MidiTrait`.

**Comment by TAdev0** [comment]
Hi @raphaelDkhn  is this issue still available ?

**Comment by raphaelDkhn** [comment]
@caseywescott are there any tests that are missing on your scope that @TAdev0 can be assigned to?

**Comment by caseywescott** [comment]
Absolutely! I'm adding more issues now but I could use help with these:

- Write tests for reverse_notes and optimize it 
- Write tests for edit_dynamics and optimize it 
- Write tests for remap_instruments and optimize it

**Comment by TAdev0** [comment]
@raphaelDkhn  should i work on any test then? the 3 previously mentioned?

**Comment by caseywescott** [comment]
@TAdev0 

I'd love it if you would work on any of those three!

**Comment by TAdev0** [comment]
I Will work on these tests then.
Not sure I can help with react or anything related to front end ..
